_posts/2022-04-01-meeting-minutes.md

```diff
@@ -0,0 +1,101 @@
+# What We're Going Over:
+- Ownership transfers
+- ACM Picnic
+- Non-computer science students outreach 
+- SIG Functional
+- Trello Board
+- Website Domain Renewal 
+
+# What We Went Over:
+- Bank Account
+    - 901.91
+    - Amanda and Carson going on the bank account
+    - Sean and Ryan going off the bank account
+- Ownerships are transfered
+- Still need to do financial transfer
+- ACM Picnic
+- Last year it was the week before dead week
+- It will be in Island Park again
+- “Get better hotdogs this time” -Sean
+- Badminton, Volleyball, and Nerf Guns
+- We need a net
+- Can we rent from the student union?
+- Get quickstart charcoal 
+- New Member at Large
+- Walmart discount
+- Will check on Sam’s Club
+- Bytle
+- Need a start date
+- Who is interested:
+    - Board members
+    - Sean
+        - Braxton
+    - Nick
+    - Gabe
+- End of year stuff to look ahead to
```

Comment:
If this is a topic, make the comments related to the topic sub bullet points (look at previous board meeting minutes as examples)